### PR TITLE
added set_bw_limits and confirm_bw intrinsic functions

### PR DESCRIPTION
**Pull request by e-schepachev**
Also get bw_limits functions return limits by actor, not summary.
Several requestbw actions supported in one transaction.

**Change Description**

**Consensus Changes**

**API Changes**

**Documentation Additions**

### CODE REVIEW

File: libraries/chain/include/eosio/chain/exceptions.hpp [hunk 1]
```diff
@@ -196,6 +196,8 @@ namespace eosio { namespace chain {
                                     3040013, "Transaction is too big" )
       FC_DECLARE_DERIVED_EXCEPTION( unknown_transaction_compression, transaction_exception,
                                     3040014, "Unknown transaction compression" )
+      FC_DECLARE_DERIVED_EXCEPTION( bandwith_confirmed, transaction_exception,
```
Comment (s-medvedev) [inline libraries/chain/include/eosio/chain/exceptions.hpp]: This name is confusing. The question appears: "If bandwidth was confirmed, why throw an exception?"
Please, rename to `bandwidth_already_confirmed`.

File: libraries/chain/transaction_context.cpp [hunk 2]
```diff
@@ -85,6 +85,28 @@ namespace bacc = boost::accumulators;
...
+   }
+
+   void provided_bandwith::set_cpu_limit(int64_t cpu_limit) {
+       verify_limits_not_confirmed();
+       this->cpu_limit_ = cpu_limit;
+   }
+
+   void provided_bandwith::verify_limits_not_confirmed() {
+       if (confirmed_) {
+           EOS_THROW( bandwith_confirmed, "bandwith has been already confirmed. No changes could be done");
```
Comment (s-medvedev) [inline libraries/chain/transaction_context.cpp]: May be `bandwidth_already_confirmed`?

Comment (afalaleev) [reply libraries/chain/transaction_context.cpp]: `EOS_ASSERT(confirmed_, bandwith_already_confirmed, '')` - looks more simple for this case.

Comment (e-schepachev) [reply libraries/chain/transaction_context.cpp]: !confirmed_

**Pull request merged**